linked-list/src/test/java/com/bobocode/LinkedListTest.java

```diff
@@ -54,22 +62,26 @@ void testGetElements() {
     }
 
     @Test
+    @Order(16)
```

Comment:
@mathodcoast I guess in this case it makes sense to **move those methods (order 6-15) here**. So all tests appear in the file in the same order they are executed.